dbms/src/Common/Stopwatch.h

```diff
@@ -23,21 +24,28 @@
 #include <common/apple_rt.h>
 #endif
 
-
-namespace StopWatchDetail
-{
-inline UInt64 nanoseconds(clockid_t clock_type)
+inline UInt64 clock_gettime_ns(clockid_t clock_type = CLOCK_MONOTONIC)
```

Comment:
besides, for `StopWatch`, how about use `std::chrono::high_resolution_clock` which provides better compatibility?

Reply:
but seems `clock_gettime` has better performance?

Reply:
sorry, `steady_clock` is what I thought. From [here](https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=libstdc%2B%2B-v3/src/c%2B%2B11/chrono.cc;h=5539d8cbedd110cd053950e86a1a4c18d4bfcf9f;hb=HEAD#l87) `steady_clock::now` also calls `clock_gettime`.

Reply:
sometimes we need better performance, we can set `clock_type_ = CLOCK_MONOTONIC`, but `steady_clock` is only `CLOCK_MONOTONIC`?